task_1/validation.js

```diff
@@ -0,0 +1,70 @@
+const init = () => {
+  let elements = document.querySelectorAll("form div");
+  for (let i = 0; i < elements.length; i++){
+    elements[i].style.display = 'none';
+  }
+}
+
+const validate = (e) => {
+  init();
+
+  let email = document.forms["form-feilds"]["email"].value;
+  let company = document.forms["form-feilds"]["company"].value;
+  let password = document.forms["form-feilds"]["password"].value;
+  let confirmPassword = document.forms["form-feilds"]["confirmPassword"].value;
+  let city = document.forms["form-feilds"]["city"].value;
+
+  const symbol = /[-!$%^&*()_+|#~=`{}\[\]:";'<>?,.\/]/;
+  const letter = /[^a-zA-Z]/;
+  const number = /\d/;
+  const emailRegEX = /^(([^<>()[\]\.,;:\s@\"]+(\.[^<>()[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i;
+
+  let isWrong = false;
+  if (company.trim() === ''){
+    document.querySelector('.company-error').style.display = 'block';
+    isWrong = true;
+  }
+
+  if (password.length < 8  || !symbol.test(password) || !letter.test(password) || !number.test(password)){
```

Comment:
Since we add the minLength to the input no need to check it here 
```suggestion
  if (!symbol.test(password) || !letter.test(password) || !number.test(password)){
```